### ISSUE TITLE
Comments on solar system science, from Renu Malhotra

### ISSUE THREAD

**Issue by michaelstrauss** [LSSTScienceCollaborations/ObservingStrategy#612]
From Renu Malhotra (a member of the SAC): 
Dear All,

Here is my homework - a review of chapter 3 and section 10.2

Overall, this is a quite comprehensive description of solar system science and strategy.  I have only some relatively minor comments.

Since LSST will be its own follow-up machine with respect to detection and confirmation and orbit linkage for minor planets, I wondered if a brief summary of that effort by the project would be useful in this chapter.

Earth minimoons are mentioned a few times and finally defined in chapter 10.2.  I did not know this terminology, probably many others don’t either.  So would be good to define it in chapter 3.

Some goals are listed in Table 3.1.  It was unclear to me why these are the specific goals.

Some simulation results are listed in Table 3.2.  It would be helpful to give a brief statement on what was assumed about the simulated populations.

In section 3.3.3, second paragraph was confusing.

Regarding “active asteroids”, I wondered if these could constitute a significant source of noise for extra-galactic science.

In section 3.5.2, second paragraph, “(mean anomaly)” should probably be “(true anomaly)”.

In section 3.6.3, first paragraph, “NEOs” should probably be “TNOs”.

Regarding light curves, there should be some consideration given to recognizing/identifying binary minor planets.

In section 10.2, there is mention of “the sweetspot”. What is meant by this? Do you mean Earth’s Trojan asteroids near 60 deg solar elongation?

In section 10.2.1, second bullet item, there is discussion of meter-size impactors, and this statement, “Impactors generally have small apparent motions on the sky (because their orbits are not too different than the Earth’s).”  Two comments: (i) I think even Earth-like orbits lead to order (1 degree per day) apparent motions. (ii) I am not aware that meter-size impactors have different orbital distribution than larger impactors, a reference would be useful here.

Typos

p. 61: “well know”  —> “well known”

p. 63: “the timing of observations effects…” —> “the timing of observations affects …”

p. 63: “The orbital uncertainties for the orbital uncertainties…” —> “The orbital uncertainties …”

p. 63: “the next ten year…” —> “the next ten years …”

p. 63: “are known to to…” —> “are known to…”

p. 64: “primordial water water ice” —> “primordial water ice”

Cheers,
Renu

**Comment by drphilmarshall** [comment]
Thanks Michael - and please do pass on our thanks to Renu for this review! 

@rhiannonlynne @davidtrilling Please take a look at Renu's comments on your Solar System chapter.  I think it should be straightforward to respond to them - perhaps you could issue the work to yourselves or others as appropriate? Thanks!

**Comment by mschwamb** [comment]
@drphilmarshall can you also assign this to me as well.  I'm taking on @rhiannonlynne's co-chair  position on the SSSC.  (thanks Lynne for your dedicated service to the SSSC)

**Comment by drphilmarshall** [comment]
Meg, I would be delighted to! I just sent you an invitation to the editors team, which has write/push access to this repo. Don't let the delay in assigning you this issue slow down your production of a pull request in response to Renu's comments! :-) Glad to have you on board - and congrats on the new position! :-)

**Comment by davidtrilling** [comment]
i'm on this as well and will coordinate with lynne. tally-ho!

david

**Comment by drphilmarshall** [comment]
That's the spirit, David! Thank you! :-)

**Comment by rhiannonlynne** [comment]
In PR #660 

--renu wrote:

"Since LSST will be its own follow-up machine with respect to detection and confirmation and orbit linkage for minor planets, I wondered if a brief summary of that effort by the project would be useful in this chapter."

David added a paragraph in S3.1 which i think might be okay, but i haven't
tried to make a "brief summary of that effort by the project". perhaps
this is something that lynne or zeljko could do (or we could just
skip it -- the rest of 3.1 does okay with describing MOPS etc.)

--references added:

Bolin et al. 2014 Icarus 241 280

--renu wrote:

"Some goals are listed in Table 3.1. It was unclear to me why these are the specific goals."

Lynne added some vague text about this where the table is referenced. The problem is that setting a required completeness level is quite hard, and is certainly not accurate to high significiant digits (80% or 85% are probably both acceptable). But on the other hand, the project needs targets to hit for the cadence + MOPS. If the cadence cannot be more than 90% effective at discovering a type of object, then MOPS had better be close to 100% - etc. So we needed some minimum level of completeness, and we came up with these goals based on getting fairly high levels of completeness for each population, down to a H magnitude that would be interesting (based on a science book, size distribution, mapping the population, etc), and saw that these goals were reasonable with the expected performance of LSST, and so these are the numbers which result.

--renu wrote:

"Some simulation results are listed in Table 3.2. It would be helpful to give a brief statement on what was assumed about the simulated populations."

Lynne added statement about these pops coming from S3M.

--renu wrote:

"In section 3.3.3, second paragraph was confusing."

Lynne simplified.

--renu wrote:

"Regarding “active asteroids”, I wondered if these could constitute a significant source of noise for extra-galactic science."

David - i would think not since active asteroids move and extragalactic
sources don't. not sure if/how to respond to this. thoughts?
Lynne - no, I don't think so, I think the extragalactic science will expect multiple observations / multiple filters in order to do something useful with the objects. 

--renu wrote:

"n section 3.6.3, first paragraph, “NEOs” should probably be “TNOs”."

David - well, it *should* be NEOs, i think, but the plot (figure 3.8)
shows no change for NEOs, but a big improvement for TNOs. 
lynne, does this make sense to you? or a problem with the figure .. ?

Lynne - so I think the text is wrong and the figure is right. I've changed it to TNOs and modified the discussion appropriately.

--typos listed by renu have apparently been 
fixed by meg schwamb.
